lib/index.ts

```diff
@@ -21,6 +22,10 @@ export * from './session/types';
 export * from './auth/types';
 export * from './webhooks/types';
 
+// Temporarily export the deprecated v5 types as a Shopify object (as opposed to the type above) to help folks find
+// the migration guide.
+export const Shopify: DeprecatedV5Types = {};
```

Comment:
This slightly conflicts with the _type_ `Shopify` we export above, but TS seems to be capable of telling them apart when importing the value.

Reply:
couldn't this be merged into the Shopify interface?

```
export interface Shopify<
  T extends ShopifyRestResources = ShopifyRestResources,
> extends DeprecatedV5Types {
  // ...
}
```